src/lib/index.js

```diff
@@ -698,10 +698,10 @@ lib.getTargetArray = function(trace, transformOpts) {
  * because extend-like algorithms are hella slow
  * obj2 is assumed to already be clean of these things (including no arrays)
  */
-lib.minExtend = function(obj1, obj2) {
+lib.minExtend = function(obj1, obj2, isPie) {
     var objOut = {};
     if(typeof obj2 !== 'object') obj2 = {};
-    var arrayLen = 3;
+    var arrayLen = isPie = 3;
```

Comment:
Not sure what happened with my editor here!
This logic is not quite right as it changes the input argument. This may slow the process.
I'll fix it and release a patch shortly.

Reply:
Fixed by #6632.